src/components/timeline/_macro-options.md

```diff
@@ -1,8 +1,8 @@
-| Name     | Type                   | Required | Description                                                                                                          |
-| -------- | ---------------------- | -------- | -------------------------------------------------------------------------------------------------------------------- |
-| classes  | string                 | false    | Classes to add to the component                                                                                      |
-| titleTag | string                 | false    | The HTML heading tag to wrap the title text in for it’s correct semantic order on the page. Will default to an `h2`. |
-| items    | array`<TimelineItems>` | true     | An array of [timeline items](#timelineitems)                                                                         |
+| Name         | Type                   | Required | Description                                                                                                                                |
+| ------------ | ---------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| classes      | string                 | false    | Classes to add to the component                                                                                                            |
+| headingLevel | int                    | false    | Number used to determine the heading level of the title. Use to ensure the title has a correct semantic order on the page. Defaults to `2` |
```

Comment:
```suggestion
| headingLevel | int                    | false    | Number used to determine the heading level of the titles. Use to ensure the title has a correct semantic order on the page. Defaults to `2` |
```
This sets the heading level for all of the titles so I think we should make this change to say that.

Also in the timelineItems table we should update the heading description because it is not always going to be a `h2`. I think we should remove `h2` from the sentence

Reply:
done